install/image-references

```diff
@@ -23,6 +23,10 @@ spec:
     from:
       kind: DockerImage
       name: registry.svc.ci.openshift.org/openshift:machine-os-content
+  - name: rhel-coreos
```

Comment:
Hmm, actually doing *this* bit blocks on https://issues.redhat.com/browse/ART-3883